hal/halx86/minihal/halinit.c

```diff
@@ -5,45 +5,22 @@
  * COPYRIGHT:   Copyright 1998 David Welch <welch@cwcom.net>
  */
 
-/* INCLUDES *****************************************************************/
+/* INCLUDES *******************************************************************/
 
 #include <hal.h>
-#define NDEBUG
-#include <debug.h>
 
-/* FUNCTIONS ***************************************************************/
+/* FUNCTIONS ******************************************************************/
```

Comment:
it's a giant waste of lifetime to groom their count to match that of other places. Such code is desperately crying for the delete-key being invoked.